bot/flat/react.py

```diff
@@ -0,0 +1,33 @@
+# coding=utf-8
+#
+import telepot
+
+from flat.members import get_members
+from flat.api_actions import update_total
+
+
+def check_text_for_command_and_execute(home, message):
+    content_type, chat_type, chat_id = telepot.glance(message)
+    try:
+        command_value = message["text"].split(" ")
+        if len(command_value) != 2:
+            raise()
+        if command_value[0].lower() != 'put':
+            raise()
+        if isinstance(command_value[1], str):
+            value = int(command_value[1])
+
+            if isinstance(value, int):
```

Comment:
this check doesn't make sense, if you are here you just did a cast to an int in the previous line